mongomock/aggregate.py

```diff
@@ -1308,7 +1308,7 @@ def _handle_project_stage(in_collection, unused_database, options):
             except KeyError:
                 # Ignore missing key.
                 pass
-    if (method == 'include') == (include_id is not False and include_id is not 0):
+    if (method == 'include') == (include_id and include_id != 0):
```

Comment:
I'm not sure this change is related. Please revert and send in a different PR if you meant it

Reply:
Ops, was a warning I was receiving. Will revert

Reply:
I agree that's annoying. I tried to silence the warning in 997adc6